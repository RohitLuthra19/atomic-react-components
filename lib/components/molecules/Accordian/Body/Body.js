// @flow
/**
 *
 * Body
 *
 */
import React from 'react';
import styled from 'styled-components';
import styles from './Body.style';

const Body = () => {
  return (
    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ut iste in laudantium voluptates
      autem blanditiis consequuntur fugit, impedit, distinctio
    </p>
  );
};

Body.defaultProps = {};

export default styled(Body)`
  ${styles};
`;

export { Body as BodyVanilla };
