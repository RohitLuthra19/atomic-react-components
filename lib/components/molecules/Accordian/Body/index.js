// @flow
export { default } from './Body';
export { BodyVanilla } from './Body';
