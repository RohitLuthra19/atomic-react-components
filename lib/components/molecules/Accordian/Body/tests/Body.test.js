import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

import Body from '../index';

describe('<Body />', () => {
  let BodyComponent = '';
  beforeEach(() => {
    BodyComponent = shallow(<Body>Test</Body>);
  });

  test('should render correctly', () => {
    expect(BodyComponent).toMatchSnapshot();
  });
});
