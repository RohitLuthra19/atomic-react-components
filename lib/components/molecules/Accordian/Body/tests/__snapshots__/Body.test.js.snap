// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> should render correctly 1`] = `
<Body
  className="Body-sc-1r9dnfr-0 jlSfwE"
>
  Test
</Body>
`;
